conanfile.txt

```diff
@@ -4,4 +4,7 @@ module_b/0.1
 
 [generators]
 CMakeToolchain
-CMakeDeps
\ No newline at end of file
+CMakeDeps
+
+[layout]
+cmake_layout
```

Comment:
Helps maintain the folder clean

Reply:
fantastic. i've been manually doing things in .gitignore...